tests/CMakeLists.txt

```diff
@@ -49,6 +46,7 @@ INCLUDE_DIRECTORIES(${CMAKE_SOURCE_DIR}/mdal/api)
 INCLUDE_DIRECTORIES(  
   ${CMAKE_CURRENT_SOURCE_DIR}
   ${CMAKE_CURRENT_BINARY_DIR}
+  ${GTEST_INCLUDE_DIR}
```

Comment:
lines 34-37 should be in if branch and GTEST_INCLUDE_DIR should be in else branch  (see travis tests failing)

```
IF(...)
  # The gtest/gmock targets carry header search path
	# dependencies automatically when using CMake 2.8.11 or
# later. Otherwise we have to add them here ourselves.
if(CMAKE_VERSION VERSION_LESS 2.8.11)
    include_directories("${gtest_SOURCE_DIR}/include"
                        "${gmock_SOURCE_DIR}/include")
endif()
ELSE(NOT GTEST_FOUND)
  include_directories( ${GTEST_INCLUDE_DIR} )
ENDIF()
```